components/ui/carousel/carousel.tsx

```
@@ -0,0 +1,87 @@
+  const videoRefs = useRef<{ [key: number]: any }>({});
+
+  const theme = useTheme();
+  const styles = stylesFn(theme);
+
+  const handleIndexChange = (index: number) => {
+    setCurrentIndex(index);
+    setIsSwiping(true);
+    // Reset swiping state after a short delay
+    setTimeout(() => setIsSwiping(false), 150);
+    Object.values(videoRefs.current).forEach((videoRef) => {
+      if (videoRef && videoRef.seek) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)